.github/components.yml

```diff
@@ -10,6 +10,7 @@ LP_transformations:
     - GPU
     - PyTorch_FE
     - TF_FE
+    - TOKENIZERS # TF_FE depends on tokenizers build
```

Comment:
Can actually be put into "build" section then, if we don't need to run **tests** for tokenizers on changes to LP transformations, only build (right now we don't have tests for tokenizers though, so it won't make much difference)

Reply:
changed